doc/chapters/design/index.rst

```diff
@@ -0,0 +1,39 @@
+SoftwareContainer Design
+************************
+
+SoftwareContainer has been designed as a building block enabling sandboxing of application contents in a larger system.
+
+The SoftwareContainer container achieves this by providing a structured way to limit what IPC mechanims are made available to each application. By targetting the IPC mechanisms rather than the applications or services directly, the sandboxing mechanism can be deployed in a non-intrusive way, i.e. completely transparent to both applications and services.
+
+Each IPC is managed by a *gateway*. The IPC needs of each service is described in *service manifest* files, defining *capabilities* and mapping these to individual *gateway configurations*. This can be individual objects or interfaces on d-bus, ipfilter rules, access to device nodes or something completely different.
```

Comment:
This is not true yet, since the component that handles capabilities has has not been integrated into SC.